Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -21,24 +22,64 @@ namespace Microsoft.Toolkit.Uwp.Helpers
     /// </summary>
     public sealed class SystemInformation
     {
-        private readonly LocalObjectStorageHelper _localObjectStorageHelper = new LocalObjectStorageHelper(new SystemSerializer());
-        private DateTime _sessionStart;
-        private PackageVersion _previousVersionInstalled;
+        /// <summary>
+        /// The <see cref="LocalObjectStorageHelper"/> instance used to save and retrieve application settings.
+        /// </summary>
+        private readonly LocalObjectStorageHelper localObjectStorageHelper = new(new SystemSerializer());
```

Comment:
Can we keep the `_` and remove the `this.` from the code? We're currently following the C# coding styles which point to the C# runtime guidelines. Though maybe that's a broader discussion. For now though let's keep consistent with what we've been doing traditionally in the Toolkit?

Reply:
Updated the code style in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/4000/commits/dd89f230e3f69e0290b07d130cb340c039aaa237 to the previous one 🙂